src/common/components/rangeDropdown/__tests__/RangeDropdown.test.tsx

```diff
@@ -0,0 +1,240 @@
+import { act, render, screen } from '@testing-library/react';
```

Comment:
Very good :) To avoid some repetition with the selectors, see how Jori has started to handle some of the queries (adding helper functions), see here:

https://github.com/City-of-Helsinki/linkedcomponents-ui/blob/feature/event-list/src/common/components/tabs/__tests__/Tabs.test.tsx

the findComponent func.